### PR TITLE
Update first node

### PR DESCRIPTION
**Pull request by leventdem**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -417,9 +417,9 @@ class Masq {
    */
   async get (key) {
     const db = this._getDB()
-    const nodes = await db.getAsync(key)
-    if (!nodes.length) return nodes[0]
-    return nodes[0].value
+    const node = await db.getAsync(key)
+    if (!node) return undefined
```
Comment (G-Ray) [inline src/index.js]: according to https://github.com/mafintosh/hyperdb/blob/master/test/basic.js#L38, we should return `null` I think

File: src/index.js [hunk 2]
```diff
@@ -417,9 +417,9 @@ class Masq {
    */
   async get (key) {
     const db = this._getDB()
-    const nodes = await db.getAsync(key)
-    if (!nodes.length) return nodes[0]
```
[the remark this answers is not in the document]
Comment (sujeebant) [reply src/index.js]: In masq-common, the createPromisified has been modified to use firstNode option

File: src/index.js (hunk 1 above)
Comment (G-Ray) [reply src/index.js]: Ca you verify that node is null ? If so, return null too

Comment (leventdem) [reply src/index.js]: yes with firstNode: true it returns null if the key does not exist.

File: src/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (leventdem) [reply src/index.js]: outdated

**Pull request merged**